File: PHP/cars/UberPool.php
<?php
    require_once('Car.php');
    class UberPool extends Car{
    private $brand;
    private $model;

    public function __construct($license,$driver,$brand,$model)
    {
        parent::__construct($license, $driver);
        $this->brand=$brand;
        $this->model = $model;
    }
    public function setModel($model){
        $this->model = $model;
    }
    public function setBrand($brand){
        $this->brand = $brand;
    }

    public function getModel(){
        return $this->model;
    }
    public function getBrand(){
        return $this->brand;
    }
        
    }
    
?>

File: pjava/cars/UberPool.java
package pjava.cars;
import pjava.account.Driver;

public class UberPool extends Car{
    private String brand;
    private String model;

    public UberPool(String license,Driver driver, String brand, String model){
        super(license,driver);
        this.brand = brand;
        this.model = model;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

}
